src/doctor.ts

```diff
@@ -0,0 +1,66 @@
+import { getProtocolInterface } from "./deps.ts";
+
+type RuntimeVersion = {
+  name: string;
+  current: string;
+  minimum?: string;
+  error?: {
+    message: string;
+  };
+};
+
+const getHostedDenoRuntimeVersion = async (): Promise<{
+  minimum?: string;
+  message?: string;
+  error?: { message: string };
+}> => {
+  try {
+    const metadataURL = "https://api.slack.com/slackcli/metadata.json";
+    const response = await fetch(metadataURL);
+    if (!response.ok) {
+      throw new Error("Failed to collect upstream CLI metadata");
```

Comment:
Let's add the error details here when we raise this, at least the HTTP response code.

Reply:
Included the status code here to hint at the error, but let me know if you think more would be better!

![status](https://github.com/slackapi/deno-slack-hooks/assets/18134219/5b9dcb3c-0ec0-4d81-837e-a2d721376f3d)

Reply:
In the same vein as my last comment, perhaps we pump the error details over the CLI/SDK protocol's diagnostic line to allow the CLI to more gracefully handle showing the error in the right place (high level error in the output, with a pointer to the debug log, where the full details can be shown)? WDYT?